Assets/Scripts/DialogueManager.cs

```diff
@@ -88,128 +127,131 @@ public void AddSpeaker(SpeakerContainer speaker)
         }
 
         speakers.Add(speaker.id, speaker);
-    }
+    }
 
 
-    public void StartConversation(Conversation convo)
+    public void StartConversation(Conversation conversation)
     {
-        // Stop any autoplay conversation
-        if (coroutine != null)
-        {
-            StopCoroutine(coroutine);
-            coroutine = null;
-        }
-
-        currentConvo = convo;
         startFrame = Time.frameCount;
 
-        if (convo.autoplayConversation)
+        // Get all speakers from the convo
+        List<string> convoSpeakers = conversation.GetAllSpeakers();
+
+        StopAllConversations(convoSpeakers);
```

Comment:
```suggestion
        StopAllConversations(conversation.GetAllSpeakers());
```
since it doesnt seem to be used after?